internal/backends/testing/search_api.rs

```diff
@@ -36,6 +36,196 @@ impl<T: ComponentHandle> ElementRoot for T {
 
 impl<T: ComponentHandle> Sealed for T {}
 
+/// Trait to obtain the initial element handle of an element tree. This is implemented for everything
+/// that implements [`ComponentHandle`].
+pub trait HasElementHandle {
```

Comment:
What's the difference with the ElementRoot trait? Why having two trait?

Reply:
Ah yes, the traits could be merged. Good idea :)